### DISCUSSION TITLE
medias can't be validated in settings section

### DISCUSSION THREAD

**Discussion by moathOSA** [area17/twill#2438] [Q&A]
i have a problem where my translated medias field can't be validated in the back-end

i tried all these roles 
'medias[banner_image][en][0]' => 'required',
'medias[banner_image][en] => 'required',
'medias.banner_image.en.0' => 'required',
'medias.banner_image.en' => 'required',
'banner_image.en' => 'required',

nothing work for me

in the rules for update and in the rules for create the same

```
public function rulesForUpdate()
    {
        return $this->rulesForTranslatedFields([
            'medias[banner_image][en][0]' => 'required',
        ], []);
    }

```

twill version 2.6.0

**Comment by ifox** [comment]
Legacy settings section do not support validation. As far as finding the right key to check for, one can take a look at the update request payload in the browser console. Medias, translated or not, need to be validated  with non translated fields rules.
